src/Directory.Build.props

```diff
@@ -198,9 +198,9 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <!-- Enable all the latest CA rules from 'Microsoft.CodeAnalysis.NetAnalyzers' as build warnings by default. Specific rules are disabled or downgraded in repo's editorconfig. -->
+    <!-- Enable performance CA rules from 'Microsoft.CodeAnalysis.NetAnalyzers' as build warnings by default. Specific rules are disabled or downgraded in repo's editorconfig. -->
     <AnalysisLevel>latest</AnalysisLevel>
-    <AnalysisMode>AllEnabledByDefault</AnalysisMode>
+    <AnalysisModePerformance>AllEnabledByDefault</AnalysisModePerformance>
```

Comment:
Analyzers other than performance analyzers may still be run, but those are ones that are enabled by default as build warning, so that's less aggressive than `<AnalysisMode>AllEnabledByDefault</AnalysisMode>`

If we wanted, we could have `<AnalysisMode>AllDisabledByDefault</AnalysisMode>` in addition to `<AnalysisModePerformance>AllEnabledByDefault</AnalysisModePerformance>`, which should hopefully by enabling exactly performance analyzers only.